app/javascript/stylesheets/page.css

```diff
@@ -14,3 +14,7 @@ body {
     flex-direction: column;
     min-height: 100vh;
 }
+
+p a {
+    @apply underline;
+}
```

Comment:
This adds an underline to all links inside of a paragraph so our styles are consistent in case we add a similar thing to the site in the future.